aws/glue/kms.tf

```
@@ -0,0 +1,74 @@
+locals {
+  glue_role_arns = [
+    aws_iam_role.glue_crawler.arn,
+    aws_iam_role.glue_etl.arn,
+  ]
+}
+
+#
+# KMS key used by AWS Glue for encryption
+#
+resource "aws_kms_key" "aws_glue" {
+  description         = "AWS Glue encryption key for data at rest"
+  enable_key_rotation = "true"
+  policy              = data.aws_iam_policy_document.aws_glue.json
+}
+
+resource "aws_kms_alias" "data_export" {
+  name          = "alias/aws-glue"
+  target_key_id = aws_kms_key.aws_glue.key_id
+}
+
+data "aws_iam_policy_document" "aws_glue" {
+  # checkov:skip=CKV_AWS_109: false-positive,`resources = ["*"]` references KMS key policy is attached to
+  # checkov:skip=CKV_AWS_111: false-positive,`resources = ["*"]` references KMS key policy is attached to
+
```

If you add this suppression comment as well, it'll clear up the last false-positive from checkov:
```suggestion
  # checkov:skip=CKV_AWS_356: `resources = ["*"]` identifies the KMS key to which the key policy is attached
```